backend/mem/fileSystem.go

```diff
@@ -103,16 +103,18 @@ func (fs *FileSystem) Scheme() string {
 //Initialize is used to initialize the fsMap for an in-memory FileSystem.
 //DISCLAIMER: nothing will work until this call is made
 
-func (fs *FileSystem) Initialize() {
+func  NewFileSystem() *FileSystem {
```

Comment:
Update NewFileSystem comments removing the space between the comment and function.